README.md

```diff
@@ -43,6 +43,21 @@ Alternatively, install in terminal:
 ## Usage
 
 You can run `falcon serve` directly. It will load the `config.ru` and start serving on https://localhost:9292.
+`serve` command has the following options that you have use it:
```

Comment:
maybe this would sound better. 

1. `serve` command has the following options at your disposal: 
2. `serve` command has the following options for you to use: